.github/workflows/gradle-check.yml

```diff
@@ -7,6 +7,10 @@ on:
       - 'dependabot/**'
   pull_request_target:
     types: [opened, synchronize, reopened]
+    paths-ignore:
```

Comment:
@gaiksaya @kotwanikunal @reta  I think this could create issues since the gradle check is required by the branch protections, you might want to be ready for a fast-follow up

Reply:
Right was thinking about that that. Wondering if GH is smart enough to detect the ignore and merge requirements conflict. Will raise a sample PR to test this theory.

Reply:
Looks like it does need the required gradle check status https://github.com/opensearch-project/OpenSearch/pull/13494
I found few discussions on GitHub community for similar issues https://github.com/orgs/community/discussions/44490
Looks like GitHub does not directly support it as of now. So some work around is required.

Reply:
Ahh. I remember why I tried using that plugin.
I was trying to set status codes instead of skipping the check completely. Doh!

Reply:
Found a work around. The solution extends @kotwanikunal approach. See https://github.com/opensearch-project/OpenSearch/pull/13498
Attached few working examples.
Thanks!